tests/test_marquez_client.py

```diff
@@ -10,483 +10,565 @@
+    'description': DESCRIPTION
+}
+
+# JOB
+JOB_NAME = 'test-job'
+JOB_ID = JobId(
+    NAMESPACE_NAME,
+    JOB_NAME
+)
+INPUTS = [DB_TABLE_ID]
+OUTPUTS = []
+LOCATION = 'https://github.com/test-org/test-repo/blob/281fe99/test_job.py'
+JOB = {
+    'id': JOB_ID,
+    'type': JobType.BATCH,
+    'name': JOB_NAME,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'namespace': NAMESPACE_NAME,
+    'inputs': INPUTS,
+    'outputs': OUTPUTS,
+    'location': LOCATION,
+    'context': None,
+    'description': DESCRIPTION,
+    'latestRun': None
+}
+
+# RUN
+NOMINAL_START_TIME = NOW_AS_ISO
+NOMINAL_END_TIME = (NOW + timedelta(days=1)).isoformat()
+START_AT = NOW
+ENDED_AT = START_AT + timedelta(hours=1)
+DURATION_MS = ((ENDED_AT - START_AT).total_seconds() * 1000)
+JOB_WITH_LATEST_RUN = {}
+RUN_ID = str(uuid.uuid4())
+NEW = {
+    'id': RUN_ID,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'nominalStartTime': NOMINAL_START_TIME,
+    'nominalEndTime': NOMINAL_END_TIME,
+    'state': RunState.NEW,
+    'startedAt': None,
+    'endedAt': None,
+    'durationMs': None,
+    'args': None
+}
+RUNNING = {
+    'id': RUN_ID,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'nominalStartTime': NOMINAL_START_TIME,
+    'nominalEndTime': NOMINAL_END_TIME,
+    'state': RunState.RUNNING,
+    'startedAt': START_AT,
+    'endedAt': None,
+    'durationMs': None,
+    'args': None
+}
+COMPLETED = {
+    'id': RUN_ID,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'nominalStartTime': NOMINAL_START_TIME,
+    'nominalEndTime': NOMINAL_END_TIME,
+    'state': RunState.COMPLETED,
+    'startedAt': START_AT,
+    'endedAt': ENDED_AT,
+    'durationMs': DURATION_MS,
+    'args': None
+}
+ABORTED = {
+    'id': RUN_ID,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'nominalStartTime': NOMINAL_START_TIME,
+    'nominalEndTime': NOMINAL_END_TIME,
+    'state': RunState.ABORTED,
+    'startedAt': START_AT,
+    'endedAt': ENDED_AT,
+    'durationMs': DURATION_MS,
+    'args': None
+}
+FAILED = {
+    'id': RUN_ID,
+    'createdAt': NOW_AS_ISO,
+    'updatedAt': NOW_AS_ISO,
+    'nominalStartTime': NOMINAL_START_TIME,
+    'nominalEndTime': NOMINAL_END_TIME,
+    'state': RunState.FAILED,
+    'startedAt': START_AT,
+    'endedAt': ENDED_AT,
+    'durationMs': DURATION_MS,
+    'args': None
+}
+
+
+@pytest.fixture
+def client():
+    return MarquezClient(url='http;//localhost:5000')
+
+
+@mock.patch('requests.put')
+def test_create_namespace(mock_put, client):
+    mock_put.return_value.status_code.return_value = 200
+    mock_put.return_value.json.return_value = NAMESPACE
+
+    namespace = client.create_namespace(
+        NAMESPACE_NAME,
+        OWNER_NAME,
+        DESCRIPTION
+    )
+
+    assert namespace['name'] == NAMESPACE_NAME
+    assert namespace['ownerName'] == OWNER_NAME
+    assert namespace['description'] == DESCRIPTION
+
+    mock_put.get.called_once()
```

Comment:
I think you want to use `assert_called_once_with` here.
Something along the lines of:
https://github.com/MarquezProject/marquez-python/blob/3995cce0b71cfa9077a878836ec77e351ced38ee/tests/test_marquez_wo_client.py#L32

Reply:
same comment for other calls bellow

Reply:
Yeah, I'll switch the assert. I was thinking maybe asserting the params was less important in the overall test. But, wouldn't hurt.